stories/VersionChooser.stories.tsx

```diff
@@ -1,16 +1,43 @@
 import * as React from 'react';
 import { storiesOf } from '@storybook/react';
 
+import configureStore from '../src/configureStore';
 import VersionChooser from '../src/components/VersionChooser';
-import { fakeVersions } from '../src/test-helpers';
+import { actions } from '../src/reducers/versions';
+import { fakeVersionsList } from '../src/test-helpers';
+import { renderWithStoreAndRouter } from './utils';
 
 storiesOf('VersionChooser', module).addWithChapters('all variants', {
   chapters: [
     {
       sections: [
         {
-          title: 'default state',
-          sectionFn: () => <VersionChooser versions={fakeVersions} />,
+          title: 'with lists of versions loaded',
+          sectionFn: () => {
+            const addonId = 124;
+
+            const store = configureStore();
+            store.dispatch(
+              actions.loadVersionsList({ addonId, versions: fakeVersionsList }),
+            );
+
+            return renderWithStoreAndRouter(
+              <VersionChooser addonId={addonId} />,
+              store,
+            );
+          },
+        },
+        {
+          title: 'loading state',
+          sectionFn: () => {
+            const addonId = 124;
```

Comment:
This makes me think these stories could benefit from a common render function that sets some defaults.

Reply:
:+1: